application/src/gui/modules/settings.py

```diff
@@ -0,0 +1,149 @@
+from PySide2.QtWidgets import QMessageBox # type: ignore
+import os
+import sys
+import toml
+import functools
+from .view_type import ViewType
+from PySide2 import QtGui  # type: ignore
+from PySide2.QtCore import Qt  # type: ignore
+from PySide2.QtWidgets import QHBoxLayout  # type: ignore
+from PySide2.QtWidgets import QPushButton  # type: ignore
+from PySide2.QtWidgets import QCheckBox  # type: ignore
+from PySide2.QtWidgets import QSpinBox  # type: ignore
+from PySide2.QtWidgets import QSlider  # type: ignore
+from PySide2.QtWidgets import QVBoxLayout  # type: ignore
+from PySide2.QtWidgets import QWidget  # type: ignore
+from PySide2.QtWidgets import QLabel  # type: ignore
+
+width = 300
+height = 220
+font = 10
+opaci_percent = 100
+opaci = opaci_percent/100
+frame = False
+top = False
+
+class FontSpin(QSpinBox):
+    def __init__(self):
+        QSpinBox.__init__(self)
+        self.setMinimum(5)
+        self.setMaximum(25)
+        self.setValue(font)
+        self.setSingleStep(1)
+
+
+class OpaciSpin(QSpinBox):
+    def __init__(self):
+        QSpinBox.__init__(self)
+        self.setMinimum(1)
+        self.setMaximum(100)
+        self.setValue(opaci_percent)
+        self.setSingleStep(10)
+        self.setSuffix("%")
+class FrameCheck(QCheckBox):
+    def __init__(self):
```

Comment:
There should be two empty lines between class definitions. `flake8` should report that.